src/components/CardPlay/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useRef, useEffect } from "react";
+import { PlayCircleTwoTone, PauseCircleTwoTone } from '@ant-design/icons';
+import "./styles.css";
+
+    const CardPlay = (props) => {
+    const audioPlayer = useRef(null);
+    const [progress, setProgress] = useState(80);
+    const [title, setTitle] = useState("Bad Bunny - Si la veo a tu mamá");
+    const [countLikes, setCountLikes] = useState(0);
+    const [countDislikes, setCountDislikes] = useState(0);
+      
+    useEffect(() => {
+      console.log(progress, audioPlayer.current.duration);
+    }, [progress]);
+    
+    const [isPlaying, setIsPlaying] = useState(false);
+    const handleLike = (event) => {
+      setCountLikes(countLikes +1)
```

Comment:
Te recomiendo estar atento a los espacios en instrucciones. Te dejo una [guía de estilos de JavaScript](https://www.w3schools.com/js/js_conventions.asp) que te puede ayudar a complementar.
```suggestion
      setCountLikes(countLikes + 1)
```